contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py

```diff
@@ -0,0 +1,446 @@
+                    (62.76901333333333, -164.50339),
+                    (62.76906333333334, -164.50353333333337),
+                ],
+                "mixed_types": [
+                    (62.75955799999999, -164.483752),
+                    ("62.7673475", "-164.4996625"),
+                    True,
+                    5,
+                    "null",
+                ],
+            },
+            "tests": [
+                {
+                    "title": "basic_positive_test",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "lat_lon",
+                        "center_point": (62.7596, -164.484),
+                        "range": 8,
+                        "unit": "miles",
+                        "projection": "fcc",
+                        "mostly": 1,
+                    },
+                    "out": {"success": True},
+                },
+                {
+                    "title": "basic_negative_test",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "lat_lon",
+                        "range": 8,
+                        "center_point": (61.7668, -162.578),
+                        "unit": "kilometers",
+                        "projection": "pythagorean",
+                        "mostly": 0.2,
+                    },
+                    "out": {"success": False},
+                },
+                {
+                    "title": "bad_type_error_test",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "mixed_types",
+                        "range": 8,
+                        "center_point": (62.7596, -164.484),
+                        "unit": "kilometers",
+                        "projection": None,
+                        "mostly": 1,
+                        "catch_exceptions": True,
+                    },
+                    "out": {},
+                    "error": {"traceback_substring": "bad operand type for unary ~:"},
+                },
+                {
+                    "title": "bad_range_parameter_input",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "lat_lon",
+                        "range": "8",
+                        "center_point": (62.7596, -164.484),
+                        "unit": "kilometers",
+                        "projection": "fcc",
+                        "catch_exceptions": True,
+                    },
+                    "out": {},
+                    "error": {"traceback_substring": "range must be a numeric value"},
+                },
+            ],
+            "only_for": "pandas",
```

Comment:
I think we want only_for to be a list, even when it's just one item

Reply:
Make's sense; resolving.